packages/perseus-editor/src/__stories__/tex-error-view.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import TexErrorView from "../tex-error-view";
+
+import type {Meta, StoryObj} from "@storybook/react";
+
+const meta: Meta<typeof TexErrorView> = {
+    component: TexErrorView,
+    title: "Perseus/Editor/TexErrorView",
+};
+
+export default meta;
+type Story = StoryObj<typeof TexErrorView>;
+
+export const Primary: Story = {
+    args: {
+        errorList: [
+            {
+                math: "x^2",
```

Comment:
Nit: if you want this story to be realistic, the math should be `"\\x^2"`. `x^2` is valid TeX.

Reply:
This was generated using CodePilot! 😫 

I'll break it to be invalid TeX.